### PR TITLE
Quotes

### PR DESCRIPTION
**Pull request by fallaciousreasoning**
Adds the ability to use the /quote <person> command (e.g. gets a quote from that person)

### CODE REVIEW

File: modules/common/wikiquote.js [hunk 1]
```diff
@@ -0,0 +1,161 @@
+var require_install = require('require-install'),
+  request = require_install('request');
+
+/**
+ * This is a *very* heavily modified version of the library found here:
+ * https://github.com/natetyler/wikiquotes-api
+ *
+ * Added by Jay Harris on 22/07/2015
+ */
+(function() {
```
Comment (mrkno) [inline modules/common/wikiquote.js]: Wow, fancy in an anonymous function and everything. You do know that node.js does this kind of scoping anyway right?

File: modules/common/wikiquote.js [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+      exports.getSectionsForPage(pageId, function(sections) { getQuotes(pageId, sections); }, errorFunction);
+    };
+
+    exports.queryTitles(titles, getSections, errorFunction);
+  };
+
+  /**
+   * Capitalize the first letter of each word
+   */
+  exports.capitalizeString = function(input) {
```
Comment (mrkno) [inline modules/common/wikiquote.js]: string.toProperCase()

Comment (mrkno) [inline modules/common/wikiquote.js]: see first few lines of main.js for things like this which are already defined

File: modules/common/wikiquote.js (hunk 1 above)
Comment (fallaciousreasoning) [inline modules/common/wikiquote.js]: It was already there. I just left it in the function :P. I can take it out if you like.

File: modules/common/wikiquote.js (hunk 2 above)
Comment (fallaciousreasoning) [inline modules/common/wikiquote.js]: Wow. The function wasn't even being used... Removed

File: modules/common/wikiquote.js (hunk 1 above)
Comment (mrkno) [inline modules/common/wikiquote.js]: Up to you

**Pull request merged**